reference/SUPPORTED_QUERIES.md

```diff
@@ -0,0 +1,159 @@
+|                         | match (EncryptedTextMatch)       | "match"              |
+| EncryptedBoolean        | eq (EncryptedOreEquals)          | "ore"                |
+| EncryptedDate           | eq (EncryptedOreEquals)          | "ore"                |
+|                         | lt (EncryptedOreLt)              | "ore"                |
+|                         | gt (EncryptedOreGt)              | "ore"                |
+| EncryptedInt            | eq (EncryptedOreEquals)          | "ore"                |
+|                         | lt (EncryptedOreLt)              | "ore"                |
+|                         | gt (EncryptedOreGt)              | "ore"                |
+| EncryptedFloat          | eq (EncryptedOreEquals)          | "ore"                |
+|                         | lt (EncryptedOreLt)              | "ore"                |
+|                         | gt (EncryptedOreGt)              | "ore"                |
+| EncryptedJsonb          | contains (EncryptedJsonContains) | "ste_vec"            |
+
+`eqldjango` also provides query expression classes:
+
+```python
+Customer.objects.filter(
+    Q(
+        CsMatch(
+            CsMatchV1(F("name")),
+            CsMatchV1(
+                Value(EqlText("ali", "customers", "name").to_db_format("match"))
+            )
+        )
+    )
+)
+```
+
+This is the more expressive but verbose method of expressing EQL queries with Django ORM.
+
+The following table shows the relevant classes in `eqldjango`.
+
+| Function or operator class | PostgreSQL function or operator |
+|----------------------------|---------------------------------|
+| CsMatchV1                  | cs_match_v1                     |
+| CsUniqueV1                 | cs_unique_v1                    |
+| CsOre648V1                 | cs_ore_64_8_v1                  |
+| CsSteVecV1                 | cs_ste_vec_v1                   |
+| CsSteVecValueV1            | cs_ste_vec_value_v1             |
+| CsSteVecTermV1             | cs_ste_vec_term_v1              |
+| CsGroupedValueV1           | cs_grouped_value_v1             |
+| CsContains                 | @>                              |
+| CsMatch                    | @>                              |
+| CsContainedBy              | <@                              |
+| CsEquals                   | =                               |
+| CsGt                       | >                               |
+| CsLt                       | <                               |
+
+### SQLAlchemy
+
+For SQLAlchemy, `eqlpy.eqlalchemy` provides python functions that correspond to the EQL functions.
+`eqlpy` also provides EQL-specific type decorators which helps with converting between the database types and Python types.
+
+```python
+session.query(Example)
+    .filter(
+        cs_match_v1(Example.encrypted_utf8_str).op("@>")(
+            cs_match_v1(
+                EqlText("hello", "examples", "encrypted_utf8_str").to_db_format(
```

Comment:
Maybe use the same table/column names as the previous example so its clear how the two approaches compare?

Reply:
Thanks for pointing this out. I copy-pasted from old examples 😓

Reply:
fixed